### ISSUE TITLE
Second level dropdown menu don't work

### ISSUE THREAD

**Issue by cotiga** [thedevdojo/voyager#4907]
## Version information
 - Laravel: 7.6
 - Voyager: 1.4
 - PHP: 7.4
 - Database: MySQL 5.7

## Description
Second submenu level don't work

## Steps To Reproduce (example)
1. Create a menu without link (like folder) in Tools menu
2. move a menu in this new submenu (screenshot 1)

## Expected behavior
In the Voyager main menu, the second submenu level don't open (screenshot 2)

## Screenshots
![capt-ecran 2020-04-27 à 20 30 11](https://user-images.githubusercontent.com/16390911/80407992-c9470b80-88c6-11ea-9377-4886a8a1cb7d.jpg)

![capt-ecran 2020-04-28 à 12 50 27](https://user-images.githubusercontent.com/16390911/80479243-01922c80-894f-11ea-99d5-5efd74d259cd.jpg)

**Comment by khodemobin** [comment]
did u got 500 error?

**Comment by cotiga** [comment]
No, when I click on an item of the 2nd level, the drop-down list is reset

**Comment by mdsEu** [comment]
@cotiga  I had the same problem, but I found that Voyager 1.4.1 has a problem too (Don't know which one), I got back to 1.3.2 version and work it

**Comment by akiyamaSM** [comment]
I'll submit a PR to fix it @cotiga

**Comment by karnwalak** [comment]
Have you solved this problem @akiyamaSM ?

**Comment by karnwalak** [comment]
Have you found any solution for this @cotiga ?
